backend/certificates.js

```diff
@@ -34,6 +34,34 @@ export function saveCertsLocally(json) {
     saveSetting('CONNECTOR_HOST_INFO', hostInfo);
     return {};
 }
+
+export function fetchCertsFromCA() {
+    // TODO: What to do when more than one user?
+    // TODO: Should I try getting the api_key if no access_token?
+    const {username, accessToken} = getSetting('USERS')[0];
+    return fetch(
+        `${CA_HOST_URL}`, {
+            method: 'POST',
+            body: JSON.stringify({credentials: {username, access_token: accessToken}})
```

Comment:
we should probably disable this for on-prem users so that their credentials don't leave the firewall